### PR TITLE
Typebating makes you sad.

### PR DESCRIPTION
**Pull request by MajManatee**
# Document the changes in your pull request
Using type bait now gives you a -1 moodlet, because you BLATANTLY caught that fish while it was talking to you.  Why would you do such a thing?

# Wiki Documentation
Add mention to the (probably nonexistent) fishing section regarding a -1 moodlet

# Changelog

:cl:  
tweak: typebaiting makes you sad.
/:cl:

### CODE REVIEW

File: code/datums/mood_events/generic_negative_events.dm [hunk 1]
```diff
@@ -221,3 +221,6 @@
 	description = "<span class='boldwarning'>HE'S CUTTING ME OPEN!!</span>\n"
 	mood_change = -8
 
+/datum/mood_event/type_bait
+	description = "<span class='warning'>I caught that fish mid-conversation... I cant belive I did that...</span>\n"
```
Comment (Moltijoe) [inline code/datums/mood_events/generic_negative_events.dm]: can't believe

**Review by Moltijoe — CHANGES_REQUESTED** [review]

File: code/datums/mood_events/generic_negative_events.dm (hunk 1 above)
Comment (Hopekz) [reply code/datums/mood_events/generic_negative_events.dm]: > can't believe

You know you can actually edit the line to let them just commit your suggestion

Comment (Moltijoe) [reply code/datums/mood_events/generic_negative_events.dm]: no, i don't

**Comment by Moltijoe** [comment]
shouldn't do anything to anyone is security or assistants, because as we know they have no conscience.

**Review by JamieD1 — APPROVED** [review]

**Comment by JamieD1** [comment]
oops, sorry molti

**Comment by Moltijoe** [comment]
> oops, sorry molti

Nah, it was more of a joke than a real suggestion

**Pull request merged**